src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionAuthorizationRequestData.java

```diff
@@ -26,6 +27,8 @@ public class TransactionAuthorizationRequestData {
     private String authorizationRequestId;
     private PaymentGateway paymentGateway;
     private URI logo;
+    @Nullable
+    private BrandType brand;
```

Comment:
I would call this cardBrand, just to make explicit that this refers only to card payments. What do you think?

Reply:
Ok no problem!